src/query.ts

```diff
@@ -119,6 +120,17 @@ export class Query {
     this.params = Object.assign({}, this.params, this._hydrate(parameters));
   }
 
+  /**
+   * Add the items dfid to the current search request, reset the page
+   * counter and set to empty the query term
+   * @param {String[]}     items            The dfid list
+   */
+  public addItems(items: string[]): void {
```

Comment:
Rename to `searchItems([…])` to follow the same name convention as for text queries (`search('something')`).